test/test_Radar.cpp

```diff
@@ -122,4 +122,14 @@ TEST_F(RadarTest,
                      heading_n2,
                      echo),
         std::runtime_error);
+}
+
+TEST_F(RadarTest, it_checks_if_measurement_is_zero)
```

Comment:
You could also test the other case

Reply:
If measurement is not zero? it is also checked here